backend/core/src/listings/listings-csv-exporter.service.ts

```diff
@@ -0,0 +1,163 @@
+        Rental_History: listing.rentalHistory,
+        Criminal_Background: listing.criminalBackground,
+        Building_Selection_Critera: listing.buildingSelectionCriteria,
+        Required_Documents: listing.requiredDocuments,
+        Important_Program_Rules: listing.programRules,
+        Special_Notes: listing.specialNotes,
+        Review_Order: listing.reviewOrderType,
+        Lottery_Date: dayjs(listing.events?.startTime).format("MM-DD-YYYY"),
+        Lottery_Start: dayjs(listing.events?.startTime).format("MM-DD-YYYY h:mm:ssA"),
+        Lottery_End: dayjs(listing.events?.endTime).format("MM-DD-YYYY h:mm:ssA"),
+        Lottery_Notes: listing.events[0]?.note,
+        Application_Due_Date: listing.applicationDueDate,
+        Waitlist: listing.isWaitlistOpen,
+        Max_Waitlist_Size: listing.waitlistMaxSize,
+        How_many_people_on_the_current_list: listing.waitlistCurrentSize,
+        How_many_open_spots_on_the_waitlist: listing.waitlistOpenSpots,
+        Marketing_Status: listing.marketingType,
+        Marketing_Season: listing.marketingSeason,
+        Marketing_Date: dayjs(listing.marketingDate).format("YYYY"),
+        Leasing_Company: listing.managementCompany,
+        Leasing_Email: listing.leasingAgentEmail,
+        Leasing_Phone: listing.leasingAgentPhone,
+        Leasing_Agent_Title: listing.leasingAgentTitle,
+        Leasing_Agent_Company_Hours: listing.leasingAgentOfficeHours,
+        Leasing_Agency_Website: listing.managementWebsite,
+        Leasing_Agency_Street_Address: listing.leasingAgentAddress?.street,
+        Leasing_Agency_Street_2: listing.leasingAgentAddress?.street2,
+        Leasing_Agency_City: listing.leasingAgentAddress?.city,
+        Leasing_Agency_Zip: listing.leasingAgentAddress?.zipCode,
+        Leasing_Agency_Mailing_Address: listing.applicationMailingAddress?.street,
+        Leasing_Agency_Mailing_Address_Street_2: listing.applicationMailingAddress?.street2,
+        Leasing_Agency_Mailing_Address_City: listing.applicationMailingAddress?.city,
+        Leasing_Agency_Mailing_Address_Zip: listing.applicationMailingAddress?.zipCode,
+        Leasing_Agency_Pickup_Address: listing.applicationPickupAddress?.street,
+        Leasing_Agency_Pickup_Address_Street_2: listing.applicationPickUpAddress?.street2,
+        Leasing_Agency_Pickup_Address_City: listing.applicationPickUpAddress?.city,
+        Leasing_Agency_Pickup_Address_Zip: listing.applicationPickUpAddress?.zipCode,
+        Leasing_Pick_Up_Office_Hours: listing.applicationPickUpAddressOfficeHours,
+        Postmark: listing.postmarkedApplicationsReceivedByDate
+          ? dayjs(listing.postmarkedApplicationsReceivedByDate).format("MM-DD-YYYY h:mm:ssA")
+          : "",
+        Digital_Application: listing.digitalApplication,
+        Digital_Application_URL: listing.applicationMethods[1]?.externalReference,
+        Paper_Application: listing.paperApplication,
+        Paper_Application_Filename:
```

Comment:
Changed from Paper_Application_URL since Paper Application doesn't allow for url option but still confirming change with product

Reply:
Clarified by product!